apps/desktop/public/electron.js

```diff
@@ -223,7 +247,10 @@ function start() {
   // This method will be called when Electron has finished its initialization and
   // is ready to create the browser windows.
   // Some APIs can only be used after this event occurs.
-  app.whenReady().then(createWindow);
+  app.whenReady().then(async () => {
+    await readAndCopyValues();
```

Comment:
Should this be called every time the app is open? 
Would it be possible to check if the user data is absent & only then copy it (so `readAndCopyValues()` only runs once)?

Will log out work properly?
We should make sure to clear both dbs on logout to avoid repopulating db with the old data

Reply:
Right now its called everytime the app is open. I believe that will rewrite old data again and again. I can change it like follows:
once the data is copied i can delete old data. Then there is no problem?

Reply:
Also consider creating a backup for old data. This allows for recovery in case the migration encounters unforeseen issues.

Reply:
> once the data is copied i can delete old data. Then there is no problem?

Good idea, @ajinkyaraj-23
However, migrating back to the older version would result in no user data - we should check with Aswin if that's something we're OK with

Reply:
Right that's a good question.